src/components/ApeTextField/ApeTextField.tsx

```diff
@@ -203,3 +140,113 @@ export const ApeTextField = ({
     </div>
   );
 };
+
+//#region Styles
```

Comment:
NIT: delete comment

Reply:
This is a region is not a comment is just a way to keep region in a file, you can colorized with some extensions in your launch

Reply:
https://github.com/microsoft/vscode/issues/12146